new-frontiers-14/frontier-station-14
# RefillableSolution: selective transfer outwards

## About the PR



Adds a field to RefillableSolutionComponent to disable outward transfer.
Removes nutriment from compressed cubes, but keeps them transferrable (they should probably also be injectable at this point but I've left that as-is)

## Why / Balance


Quasi-hacky fix for https://github.com/space-wizards/space-station-14/issues/33115.

Pizza sanctity.

Soup should probably be an exception, but a differentiated SoupBase vs. FoodBowlBase would also be useful here.

## How to test


1. Drag a pizza onto a ChemMaster, nothing happens.
2. Pour lime juice onto a monkey cube, drag it into the ChemMaster, 1u of lime juice comes out.
3. Spawn a mop, spawn a water bottle, and something to make a puddle.  Mop up the water, mop up the puddle, spawn a sausage bread.  Wring out the mop into the sausage bread.  The not-water should be transferred.  Bone apple teeth.
4. Spawn a drain, try to drain the soggy sausage bread, nothing should happen.

## Media


Just like mama used to make.
![image](https://github.com/user-attachments/assets/fc843489-6921-429f-93df-b3b65586f637)

## Requirements



## Breaking changes


**Changelog**


:cl:
- fix: Reagents in food solutions cannot be transferred out, e.g. into a ChemMaster.
